.github/workflows/integration-tests.yaml

```diff
@@ -96,3 +96,15 @@ jobs:
           python3 -m pip install -e sdk/python; pytest sdk/python/test --log-cli-level=info --namespace=default
         env:
           GANG_SCHEDULER_NAME: ${{ matrix.gang-scheduler-name }}
+
+      - name: Collect volcano logs
+        if: ${{ failure() &&  env.GANG_SCHEDULER_NAME == 'volcano' }}
```

Comment:
`GANG_SCHEDULER_NAME` is only set to the task, `Deploy training operator`. So is it possible to read`GANG_SCHEDULER_NAME` here?

Reply:
Have changed to `matrix.gang-scheduler-name`.